pkgs/tools/admin/swiftclient/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildPythonApplication, fetchPypi, requests, six, pbr }:
+
+buildPythonApplication rec {
+  pname = "python-swiftclient";
+  version = "3.6.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0sv6z72zdwzwdjng0djk3l2maryn9pz3khf69yq5ig2ycz8hh0qv";
+  };
+
+  propagatedBuildInputs = [ requests six pbr ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = https://github.com/openstack/python-swiftclient;
+    description = "Python bindings to the OpenStack Object Storage API";
+    license = licenses.apache;
```

Comment:
Meta checks are failing because there is no stdenv.lib.licenses.apache. You probably meant asl20. :)

Reply:
Ah yes, thanks that's the one. Weird that my local build succeeded with this error in it.

Reply:
It's not weird (unfortunately) since a normal nix-build doesn't even touch the meta attributes (aside, I think, from checking for unfree licenses)